external-docs/docs/guides/shape-data.md

```diff
@@ -1,19 +1,17 @@
 # Shaping your data
 
-> This page discusses the shapes of input data supported by csvcubed with the aim to help you understand which to use and how to generate data in the right shape.  
+> This page introduces the input data shapes supported by csvcubed with the aim of helping you understand which to use and how to generate data in the right shape.  
 
-csvcubed requires that all CSV data inputs are provided in one of two specialised form of [tidy data](../glossary/index.md#tidy-data):
+csvcubed requires that all CSV data inputs are provided in one of two specialised forms of [tidy data](../glossary/index.md#tidy-data):
 
-* the [standard approach](#standard-shape) - the **default recommended shape** accepted by csvcubed. It is the **most flexible** approach as it allows you to vary measures and units within the same cube, but it is also the most **verbose** shape.
-* the [pivoted approach](#pivoted-shape) - a **terser** shape currently only compatible with data sets containing a **single measure**.
+* the [standard approach](./standard-shape.md) - the recommended shape for sparse multi-measure data cubes.
```

Comment:
Can you link to the `sparse` and `dense` definitions in the glossary here?

Reply:
Done